Validator/Validator.cs

```diff
@@ -112,5 +111,15 @@ public static bool Equals(string input, string comparison)
         {
             return input.Equals(comparison);
         }
+
+        public static bool IsDate(string input)
+        {
+            if (String.IsNullOrEmpty(input))
```

Comment:
Can you please write a test that passes an empty string?

Can you also verify that this check is needed? I am curious why `TryParse` does when you supply an empty string.